### ISSUE TITLE
Documentation: wrong syntax description

### ISSUE THREAD

**Issue by grossjohann** [mookid/diffr#40]
`diffr --help` prints this:

```
            color_spec = face-name + ':' + attributes
            attributes = <empty>
                       | attribute + ':' + attributes
```

This means that a `color_spec` always ends with a colon, but I'm sure this wasn't intended.  Maybe reword like this:

```
            color_spec = face-name + ':' + attributes
            attributes = attribute
                       | attribute + ':' + attributes
```

This is more restrictive than the original grammar because it requires at least one attribute.  I don't know if that matches the implementation.  But it would be easy to fix by allowing `color_spec` to expand to either a bare `face-name` or to `face-name + ':' + attributes`.

**Comment by mookid** [comment]
Great catch! The doc change you suggest is definitely what I had in mind.  Would you submit a PR for that?

**Comment by mookid** [comment]
thanks!

**Comment by grossjohann** [comment]
Thank you very much!
